health-checker/src/main/java/io/stargate/health/StorageHealthChecker.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.health;
+
+import com.codahale.metrics.health.HealthCheck;
+import com.datastax.oss.driver.api.core.uuid.Uuids;
+import io.stargate.db.datastore.DataStore;
+import io.stargate.db.datastore.DataStoreFactory;
+import io.stargate.db.datastore.ResultSet;
+import io.stargate.db.datastore.Row;
+import io.stargate.db.query.Predicate;
+import io.stargate.db.query.builder.Replication;
+import io.stargate.db.schema.Column;
+import io.stargate.db.schema.ImmutableColumn;
+import io.stargate.db.schema.ImmutableTable;
+import io.stargate.db.schema.Table;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class StorageHealthChecker extends HealthCheck {
+  private static final Logger logger = LoggerFactory.getLogger(DataStoreHealthChecker.class);
+  private static final boolean SHOULD_CREATE_KS_AND_TABLE =
+      Boolean.parseBoolean(
+          System.getProperty("stargate.health_check.data_store.create_ks_and_table", "true"));
+  private static final String KEYSPACE_NAME =
+      System.getProperty("stargate.health_check.keyspace_name", "data_store_health_check");
+  private static final String TABLE_NAME =
+      System.getProperty("stargate.health_check.table_name", "health_table");
+  private static final int INSERT_TTL_SECONDS = 600;
```

Comment:
This seems really high considering most health checks will run several times a minute.

Reply:
I suggested bumping it up based on slow schema sync observations in smaller environments... how about 180 (3 min)?

Reply:
I think that after reworking it to one row per stargate node (https://github.com/riptano/floodgate/compare/main...stargate_number_of_connections?expand=1) it's totally safe to have the TTL for 10 minutes. 
When storage is operational, the record for the stargate node will be overridden. In such a scenario, we should have `number of rows == number of Stargate nodes`
The ttl would be needed only when the stargate node will be recreated. If one stargate node is re-created, we will have N + 1 rows in the table (`N == number of stargate nodes`) for TTL time. Assuming that this is not a very common situation, the higher TTL should be ok.